dev-docs/howto/vpn/README.md

```diff
@@ -0,0 +1,36 @@
+# Constellation VPN
+
+This Helm chart deploys a VPN server to your Constellation cluster.
+
+## Installation
+
+1. Create and populate the configuration.
+
+   ```sh
+   helm inspect values . >config.yaml
+   ```
+
+2. Install the Helm chart.
+
+   ```sh
+   helm install vpn . >config.yaml
```

Comment:
Overwriting the config looks wrong. Did you mean to
```suggestion
   helm install -f config.yaml vpn . 
```

Reply:
Good catch, thanks.